.env

```diff
@@ -24,3 +24,4 @@ SUPPORT_EMAIL=''
 SUPPORT_URL=''
 USER_INFO_COOKIE_NAME=''
 ENABLE_PROGRESS_GRAPH_SETTINGS=false
+ENABLE_TEAM_TYPE_SETTING=false
```

Comment:
This will need to be set in edx-internal repo as well?

Reply:
Yes, but we will be doing that in a followup ticket when we need to re-enable this field for teams!
Right now for rollout, we will get these fallback values, and as a result hide the `TYPE` field for teamset!